src/System.Management.Automation/engine/parser/Parser.cs

```diff
@@ -2744,28 +2744,42 @@ private StatementAst SwitchStatementRule(LabelToken labelToken, Token switchToke
 
                 while (true)
                 {
-                    ExpressionAst clauseCondition = GetSingleCommandArgument(CommandArgumentContext.SwitchCondition);
-                    if (clauseCondition == null)
-                    {
-                        // ErrorRecovery: if we don't have anything that looks like a condition, we won't
-                        // find a body (because a body is just a script block, which works as a condition.)
-                        // So don't look for a body, hope we find the '}' next.
+                    Token token = PeekToken();
+                    var isDefaultClause = token.Kind == TokenKind.Default;
+                    ExpressionAst clauseCondition = null;
 
-                        isError = true;
-                        ReportIncompleteInput(After(endErrorStatement),
-                            nameof(ParserStrings.MissingSwitchConditionExpression),
-                            ParserStrings.MissingSwitchConditionExpression);
-                        // Consume a closing curly, if there is one, to avoid an extra error
-                        if (PeekToken().Kind == TokenKind.RCurly)
+                    if (isDefaultClause)
+                    {
+                        // Consume the 'default' token.
+                        SkipToken();
+                        endErrorStatement = token.Extent;
+                    }
+                    else
+                    {
+                        clauseCondition = GetSingleCommandArgument(CommandArgumentContext.SwitchCondition);
+                        if (clauseCondition == null)
                         {
-                            SkipToken();
+                            // ErrorRecovery: if we don't have anything that looks like a condition, we won't
+                            // find a body (because a body is just a script block, which works as a condition.)
+                            // So don't look for a body, hope we find the '}' next.
+
```

Comment:
Extra line.
```suggestion
```

Reply:
I did not add this extra line, I only indented the code block within another `if` block.   Do you still want the line removed?

Reply:
CodeFactor will complains on your commit. So please remove.